financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupScreen.kt

```diff
@@ -84,181 +88,165 @@ private fun NetworkingLinkLoginWarmupContent(
+            footer = {
+                Footer(
+                    disableNetworkingAsync = disableNetworkingAsync,
+                    onContinueClick = onContinueClick,
+                    onSkipClicked = onSkipClicked
+                )
+            }
         )
-        Spacer(modifier = Modifier.size(20.dp))
-        SkipSignIn(onClickableTextClick)
     }
 }
 
 @Composable
-private fun SkipSignIn(onClickableTextClick: (String) -> Unit) {
-    AnnotatedText(
-        text = StringId(R.string.stripe_networking_link_login_warmup_skip),
-        defaultStyle = FinancialConnectionsTheme.typography.caption.copy(
-            color = FinancialConnectionsTheme.colors.textSecondary
-        ),
-        annotationStyles = mapOf(
-            StringAnnotation.CLICKABLE to FinancialConnectionsTheme.typography.captionEmphasized
-                .toSpanStyle()
-                .copy(color = FinancialConnectionsTheme.colors.textBrand),
-        ),
-        onClickableTextClick = onClickableTextClick,
-    )
-}
-
-@Composable
-private fun Description(onClickableTextClick: (String) -> Unit) {
-    AnnotatedText(
-        text = StringId(R.string.stripe_networking_link_login_warmup_description),
-        defaultStyle = FinancialConnectionsTheme.typography.body.copy(
-            color = FinancialConnectionsTheme.colors.textSecondary
-        ),
-        annotationStyles = mapOf(
-            StringAnnotation.CLICKABLE to FinancialConnectionsTheme.typography.body
-                .toSpanStyle()
-                .copy(color = FinancialConnectionsTheme.colors.textBrand),
-        ),
-        onClickableTextClick = onClickableTextClick,
-    )
-}
-
-@Composable
-private fun Title(onClickableTextClick: (String) -> Unit) {
-    AnnotatedText(
-        text = StringId(R.string.stripe_networking_link_login_warmup_title),
-        defaultStyle = FinancialConnectionsTheme.typography.subtitle,
-        annotationStyles = mapOf(
-            StringAnnotation.CLICKABLE to FinancialConnectionsTheme.typography.subtitle
-                .toSpanStyle()
-                .copy(color = FinancialConnectionsTheme.colors.textBrand),
-        ),
-        onClickableTextClick = onClickableTextClick,
-    )
+@OptIn(ExperimentalComposeUiApi::class)
+private fun Footer(
+    disableNetworkingAsync: Async<FinancialConnectionsSessionManifest>,
+    onContinueClick: () -> Unit,
+    onSkipClicked: () -> Unit
+) {
+    Column {
+        FinancialConnectionsButton(
+            loading = false,
+            enabled = disableNetworkingAsync !is Loading,
+            type = Type.Primary,
+            onClick = onContinueClick,
+            modifier = Modifier
+                .semantics { testTagsAsResourceId = true }
+                .testTag("existing_email-button")
+                .fillMaxWidth()
+        ) {
+            Text(text = stringResource(id = R.string.stripe_networking_link_login_warmup_cta_continue))
+        }
+        Spacer(modifier = Modifier.size(16.dp))
```

Comment:
Nit: Could be with `Arrangement.spacedBy`.

Reply:
cleaned up spacing here: https://github.com/stripe/stripe-android/pull/7830/commits/83321149c01c6cce7153f29b58ace57f76946c42